src/components/SummaryManagement/SummaryReceiverPopup.jsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable react/prop-types */
+import React, { useState, useEffect } from 'react';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter, Container, Alert } from 'reactstrap';
+import MembersAutoComplete from './MembersAutoComplete';
+//import hasPermission from 'utils/permissions';
+
+const TeamMembersPopup = React.memo(props => {
+  // debugger;
+  const closePopup = () => {
+    props.onClose();
+  };
+  const [selectedUser, onSelectUser] = useState(undefined);
+  const [isValidUser, onValidation] = useState(true);
+  const [searchText, setSearchText] = useState('');
+
+  const onAddUser = () => {
+    cd;
```

Comment:
This looks like a typo and removing it seems to make the receiver popup work as intended.
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/6c5cdc6f-d392-49aa-9cee-6ada27cc204a)
```suggestion
```

Reply:
delete typo erro , please check thanks.